app/assets/stylesheets/modules/_family.scss

```diff
@@ -123,12 +125,13 @@
 }
 
 .family-section-three-image {
-  @include span-columns(10 of 10);
   background: image-url("faces.png");
   background-repeat: no-repeat;
-  background-size: 100%;
+  background-size: em(282);
+  width: em(282);
```

Comment:
Properties should be ordered background, background-repeat, background-size, display, height, margin, width